content/en/agent/logs/log-transport.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Agent Transport for logs
+kind: documentation
+description: Use the Datadog Agent to collect your logs and send them to Datadog
+further_reading:
+- link: "agent/logs/advanced_log_collection/#filter-logs"
+  tag: "Documentation"
+  text: "Filter logs sent to Datadog"
+- link: "agent/logs/advanced_log_collection/#scrub-sensitive-data-from-your-logs"
+  tag: "Documentation"
+  text: "Scrub sensitive data from your logs"
+- link: "agent/logs/advanced_log_collection/#multi-line-aggregation"
+  tag: "Documentation"
+  text: "Multi-line log aggregation"
+- link: "agent/logs/advanced_log_collection/#tail-directories-by-using-wildcards"
+  tag: "Documentation"
+  text: "Tail directories by using wildcards"
+- link: "agent/logs/advanced_log_collection/#global-processing-rules"
+  tag: "Documentation"
+  text: "Global processing rules"
+---
+
+
+## Default agent behavior
+
+Since Agent v6.19+/v7.19+, the default transport is HTTPS with compression instead of TCP.
+At agent start, if log collection is enabled, the agent runs a HTTPS connectivity test:
+
+* if successful, then the agent uses compressed HTTPS transport
+* otherwise the agent fallbacks to TCP transport
```

Comment:
```suggestion
When the Agent starts, if log collection is enabled, it runs a HTTPS connectivity test. If successful, then the Agent uses the compressed HTTPS transport, otherwise the Agent fallbacks to a TCP transport.
```